inst/extdata/child/intro-emissions_profile.Rmd

```diff
@@ -1,35 +1,41 @@
-The "emissions profile" measures the transition risk on the product-level of a
-company. The indicator is expressed in the percentage of all products that are
-at high risk, medium risk or low risk due to the products' relative greenhouse
-gas (GHG) emissions. The assessment is first performed on a product-level and
-can then be aggregated to the company-level.
+The "emissions profile" measures the absolute GHG emissions of a product in
+comparison to a chosen benchmark of products. The assessment is first performed
+on a product-level and can then be aggregated to the company-level. The profile
+is expressed as the share of a company's products that are in a "high",
+"medium", or "low" emission profile category, based on the comparison to the
+benchmark. A higher emission profile indicates a larger impact on climate change
+compared to the benchmark and can therefore also be interpreted as one climate
+risk component to assess products, companies, or loan portfolios.
 
-The "emissions profile" measures the relative GHG emissions per product. As a
-default option, each product is compared to the GHG emissions of every other
-product. Products with higher GHG emissions are considered to have a higher
-emission profile and hence also face a higher climate risk.
+As a first step to calculate the "emission profile", the relative GHG emissions
+per product are collected from a Life-Cycle-Analysis (LCA) database by matching
+the products from our company dataset to the products from the LCA dataset. As a
+second step, all products are ranked according to their GHG emissions. Thirdly,
+the products are grouped by the following 6 benchmarks:
 
-After identifying the GHG emissions for each product, the products are ranked
-according to their carbon footprint. The ranking method is explained in more
-detail in the
-[Thresholds](https://2degreesinvesting.github.io/tiltIndicator/articles/tiltIndicator.html#thresholds)
-section. In short, products with relatively low GHG emissions are assigned to
-the category "low", products with relatively high GHG emissions to the category
-"high" and the products in between to the category "medium".
+* "all": All products.
+* "unit": All products with the same unit (example: kg).
+* "tilt\_sec": All products within the same tilt sector (example: agriculture).
+* "isic\_4digit": All products within the same ISIC 4 digit code (example:
+0112 Growing of rice).
+* "unit\_tilt\_sec": All products with the same unit within the same tilt sector
+(example: kg + agriculture).
+* "unit\_isic": All products with the same unit within the same ISIC 4 digit
+section (example: kg + 0112 Growing of rice).
```

Comment:
Like in code-comments, it's best to avoid saying what the code exactly does. When the code changes we have no automated way to find all the text that needs to be updated. The result is that the text no longer reflects what the code does and becomes confusing.

And this is a good example, as this text is no longer correct. 

See https://2degreesinvesting.github.io/tiltIndicator/news/index.html#tiltindicator-0009095

> passing a column `products$tilt_sector` now yields the value “tilt_sector” in `group_by` – which before got cropped to `“tilt_sec”`.

``` r
library(tidyverse)
library(tiltIndicator)
library(tiltToyData)

options(readr.show_col_types = FALSE)

companeis <- read_csv(toy_emissions_profile_any_companies())
products <- read_csv(toy_emissions_profile_products_ecoinvent())

result <- emissions_profile(companies, products)
#> Warning: `companies` was deprecated in tiltIndicator 0.0.0.9089. Please use
#> `emissions_profile_any_companies` from tiltToyData.

result |> 
  unnest_product() |> 
  distinct(grouped_by)
#> # A tibble: 6 × 1
#>   grouped_by      
#>   <chr>           
#> 1 all             
#> 2 isic_4digit     
#> 3 tilt_sector     
#> 4 unit            
#> 5 unit_isic_4digit
#> 6 unit_tilt_sector

result |> 
  unnest_company() |>
  distinct(grouped_by)
#> # A tibble: 6 × 1
#>   grouped_by      
#>   <chr>           
#> 1 all             
#> 2 isic_4digit     
#> 3 tilt_sector     
#> 4 unit            
#> 5 unit_isic_4digit
#> 6 unit_tilt_sector
```

<sup>Created on 2024-01-18 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>